frontend/javascripts/test/backend-snapshot-tests/__snapshots__/projects.e2e.ts.snap

```diff
@@ -555,559 +555,3 @@ exports[`Project API (E2E) > updateProject(projectId: string, project: APIProjec
   "teamName": "team_X1",
 }
 `;
-
-exports[`Project API > createProject and deleteProject 1`] = `
```

Comment:
The test suite was renamed from `Project API` to `Project API (E2E)`. Hence, the snapshots were marked and reported as `obsolete` by `vitest`.